tensorflow/lite/micro/kernels/tanh_test.cc

```diff
@@ -67,6 +65,93 @@ const float tanh_output_vec_fp[tanh_vec_size] = {
+    -18.1818181818, -17.9545454545, -17.7272727273, -17.5000000000,
+    -17.2727272727, -17.0454545455, -16.8181818182, -16.5909090909,
+    -16.3636363636, -16.1363636364, -15.9090909091, -15.6818181818,
+    -15.4545454545, -15.2272727273, -15.0000000000, -14.7727272727,
+    -14.5454545455, -14.3181818182, -14.0909090909, -13.8636363636,
+    -13.6363636364, -13.4090909091, -13.1818181818, -12.9545454545,
+    -12.7272727273, -12.5000000000, -12.2727272727, -12.0454545455,
+    -11.8181818182, -11.5909090909, -11.3636363636, -11.1363636364,
+    -10.9090909091, -10.6818181818, -10.4545454545, -10.2272727273,
+    -10.0000000000, -9.7727272727,  -9.5454545455,  -9.3181818182,
+    -9.0909090909,  -8.8636363636,  -8.6363636364,  -8.4090909091,
+    -8.1818181818,  -7.9545454545,  -7.7272727273,  -7.5000000000,
+    -7.2727272727,  -7.0454545455,  -6.8181818182,  -6.5909090909,
+    -6.3636363636,  -6.1363636364,  -5.9090909091,  -5.6818181818,
+    -5.4545454545,  -5.2272727273,  -5.0000000000,  -4.7727272727,
+    -4.5454545455,  -4.3181818182,  -4.0909090909,  -3.8636363636,
+    -3.6363636364,  -3.4090909091,  -3.1818181818,  -2.9545454545,
+    -2.7272727273,  -2.5000000000,  -2.2727272727,  -2.0454545455,
+    -1.8181818182,  -1.5909090909,  -1.3636363636,  -1.1363636364,
+    -0.9090909091,  -0.6818181818,  -0.4545454545,  -0.2272727273,
+    0.0000000000,   0.2272727273,   0.4545454545,   0.6818181818,
+    0.9090909091,   1.1363636364,   1.3636363636,   1.5909090909,
+    1.8181818182,   2.0454545455,   2.2727272727,   2.5000000000,
+    2.7272727273,   2.9545454545,   3.1818181818,   3.4090909091,
+    3.6363636364,   3.8636363636,   4.0909090909,   4.3181818182,
+    4.5454545455,   4.7727272727,   5.0000000000,   5.2272727273,
+    5.4545454545,   5.6818181818,   5.9090909091,   6.1363636364,
+    6.3636363636,   6.5909090909,   6.8181818182,   7.0454545455,
+    7.2727272727,   7.5000000000,   7.7272727273,   7.9545454545,
+    8.1818181818,   8.4090909091,   8.6363636364,   8.8636363636,
+    9.0909090909,   9.3181818182,   9.5454545455,   9.7727272727,
+    10.0000000000,  10.2272727273,  10.4545454545,  10.6818181818,
+    10.9090909091,  11.1363636364,  11.3636363636,  11.5909090909,
+    11.8181818182,  12.0454545455,  12.2727272727,  12.5000000000,
+    12.7272727273,  12.9545454545,  13.1818181818,  13.4090909091,
+    13.6363636364,  13.8636363636,  14.0909090909,  14.3181818182,
+    14.5454545455,  14.7727272727,  15.0000000000,  15.2272727273,
+    15.4545454545,  15.6818181818,  15.9090909091,  16.1363636364,
+    16.3636363636,  16.5909090909,  16.8181818182,  17.0454545455,
+    17.2727272727,  17.5000000000,  17.7272727273,  17.9545454545,
+    18.1818181818,  18.4090909091,  18.6363636364,  18.8636363636,
+    19.0909090909,  19.3181818182,  19.5454545455,  19.7727272727,
+    20.0000000000};
+
+const float tanh_int16_output_vec_fp[tanh_int16_vec_size] = {
```

Comment:
nit: If possible, add some comments on where the golden data comes from?

Reply:
Done.